docs/_src/usage/usage/knowledge_graph.md

```diff
@@ -0,0 +1,108 @@
+Two classes implement the functionalities: GraphDBKnowledgeGraph and Text2SparqlRetriever.
+
+## GraphDBKnowledgeGraph
+
+GraphDBKnowledgeGraph is a triple store similar to Haystack's document stores. Currently, it is the only implementation of the BaseKnowledgeGraph class.
+GraphDBKnowledgeGraph runs on GraphDB. The licensing of GraphDB is rather complicated and it's more than unfortunate that GraphDB cannot be used right away in colab notebooks.
+On your local machine, you can start a GraphDB instance by running:
+
+```docker run -d -p 7200:7200 --name graphdb-instance-tutorial docker-registry.ontotext.com/graphdb-free:9.4.1-adoptopenjdk11```
+
+By default, GraphDBKnowledgeGraph connects to a GraphDB instance running on localhost at port 7200. 
+Similar to Haystack's ElasticsearchDocumentStore, the only additional setting needed is an index name.
+(Note that GraphDB internally calls these indices repositories.)
+
+```kg = GraphDBKnowledgeGraph(index="tutorial_10_index")```
+
+Indices can be deleted and created with ```delete_index()``` and ```create_index(config_path)```.
+```config_path``` needs to point to a .ttl file that contains configuration settings (see [GraphDB documentation](https://graphdb.ontotext.com/documentation/free/configuring-a-repository.html#configure-a-repository-programmatically) for details or use the file from our [tutorial](https://github.com/deepset-ai/haystack/blob/master/tutorials/Tutorial10_Knowledge_Graph.py)). It starts with something like:
+
+```
+#
+# Sesame configuration template for a GraphDB Free repository
+#
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix rep: <http://www.openrdf.org/config/repository#>.
+@prefix sr: <http://www.openrdf.org/config/repository/sail#>.
+@prefix sail: <http://www.openrdf.org/config/sail#>.
+@prefix owlim: <http://www.ontotext.com/trree/owlim#>.
+
+[] a rep:Repository ;
+    rep:repositoryID "tutorial_10_index" ;
+    rdfs:label "tutorial 10 index" ;
+...
+```
+
+GraphDBKnowledgeGraph can load an existing knowledge graph represented in the form of a .ttl file with the method ```import_from_ttl_file(index, path)```, where path points to a ttl file starting with something like:
+
+```
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix hp: <https://deepset.ai/harry_potter/> .
+
+hp:Gryffindor hp:source_url "https://harrypotter.fandom.com/wiki/Gryffindor"^^xsd:string .
+hp:Gryffindor rdf:type hp:House_ .
+hp:Gryffindor hp:name hp:Gryffindor .
+hp:Gryffindor hp:founder hp:Godric_gryffindor .
+...
+```
+
+```get_all_triples()``` returns all loaded triples in the form of subject, predicate, and object. It is helpful to check whether the loading of a .ttl file was successful.
```

Comment:
I'd suggesting we write out method calls as `GraphDBKnowledgeGraph.get_all_triples()` so its unambiguous which class it is a part of

Reply:
Thanks for the suggestion. I have changed the method calls accordingly. 👍